mobile/library/java/io/envoyproxy/envoymobile/engine/types/BUILD

```diff
@@ -1,11 +1,11 @@
+load("@build_bazel_rules_android//android:rules.bzl", "android_library")
 load("@envoy//bazel:envoy_build_system.bzl", "envoy_mobile_package")
-load("@rules_java//java:defs.bzl", "java_library")
 
 licenses(["notice"])  # Apache 2
 
 envoy_mobile_package()
 
-java_library(
+android_library(
```

Comment:
if we want to make these tests non-mobile specific (i.e. they can run on other platforms as a client networking library), should we instead standardize on java_library and kotlin_library or something similar?

Reply:
Currently the code is kinda messy and it's not easy to do that. If you look at Cronvoy, everything in there is `android_library`. So, this change is to make it consistent.

In the future, if we want to separate non-Android specific libraries, we should be putting the Android specific stuff under `android` directory instead of mixing the Android and non-Android files, which makes it very easy to make a mistake.

At some point in the future, I may refactor / rewrite everything, so it's probably not worth splitting the Android and non-Android libraries.